### ISSUE TITLE
problem to add setSuggestions

### ISSUE THREAD

**Issue by sepehr-alipour** [MiguelCatalan/MaterialSearchView#27]
when i add this code      searchView.setSuggestions(getResources().getStringArray(R.array.query_suggestions));
search view fill other views of  app on runtime but by tapping on search button and press back other views showing

**Comment by MiguelCatalan** [comment]
I'm so sorry but, can you rephrase it?

I don't really get what is the problem.

**Comment by sepehr-alipour** [comment]
This problem only exists in new version(1.3.0)
run app with including this code 
i have tabs under toolbar and
when i add this code searchView.setSuggestions(getResources().getStringArray(R.array.query_suggestions));
all views disappear like this
![screenshot_20151027-211607](https://cloud.githubusercontent.com/assets/6023851/10767645/4b5d1794-7cf2-11e5-81bf-aa8284e32691.png)



but by tapping on search icon we have this screen

![screenshot_20151027-211611](https://cloud.githubusercontent.com/assets/6023851/10767602/15ae4636-7cf2-11e5-80be-c79776b2bea4.png)


after tap on back button all view appear like this
![screenshot_20151027-211619](https://cloud.githubusercontent.com/assets/6023851/10767625/32e9f948-7cf2-11e5-8094-949fa30b4f4e.png)


by removing that code all thing work correctly or in older version
I hope you will understand what I mean

**Comment by MiguelCatalan** [comment]
Ok I will check it.

I have not much time right now, so I can't tell you when it will be fixed. :confused:

**Comment by sepehr-alipour** [comment]
I solved my issue by chang the code:

@Override 
public void onSearchViewShown() { searchView.setSuggestions(getResources().getStringArray(R.array.query_suggestions));
}

**Comment by MiguelCatalan** [comment]
Cool, thanks!

**Comment by revolz** [comment]
@MiguelCatalan, Do you think you are able to take a look at this bug again? Thanks.
@sepehr-alipour, Thanks for your answer [setSuggestions() in onSearchViewShown()], it gives a workaround. But I wonder if this will add more computing load as it will run for every event of search view shown?

**Comment by Ehsanfb** [comment]
to solve the problem, you should omit android:layout_below="@id/toolbar_container" from below layout (for example a recyclerview) and replace android:layout_marginTop="?attr/actionBarSize"
